.kitchen.yml

```diff
@@ -17,16 +17,27 @@ platforms:
 suites:
 - name: delete_from_list
   run_list:
+    - recipe[test::resource_status]
```

Comment:
We could just split these tests out into their own suite. Rather than adding this on? 

Whilst it's quite nice. It's hard to get at/debug for a beginner.